bin/mocha-current-file.js

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env node
+
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/build
```

Comment:
Let's fix the headers.

Reply:
...done?  It's not consistent in the files in this directory.